core/drivers/crypto/caam/ae/local.h

```diff
@@ -0,0 +1,78 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2024 NXP
+ */
+#ifndef __LOCAL_H__
+#define __LOCAL_H__
+
+#include <caam_utils_dmaobj.h>
+#include <drvcrypt.h>
+#include <drvcrypt_authenc.h>
+
+/* Maximum AAD size */
+#define AAD_LENGTH_OVERFLOW 0xFF00
+
+/*
+ * Cipher Algorithm definition
+ */
+struct cipheralg {
+	uint32_t type; /* Algo type for operation */
```

Comment:
When documenting a struct we often prefer doing like:
https://github.com/OP-TEE/optee_os/blob/f0489baa765d4c0f88ad047b5e4d5120907161f1/core/include/mm/fobj.h#L31-L50